### PR TITLE
TL improvement

### PR DESCRIPTION
**Pull request by msksk**
I would like to check TLs in Kotohogushi after completing Miotsukushi.

### CODE REVIEW

**Comment by DoctorDiablo** [comment]
Please wait a little while before touching Kotohogushi. I need to do some stuff to the script files and I was planning to do a review of my own. I want to avoid dealing with merge conflicts from 3 people editing the same files at the same time.

**Comment by msksk** [comment]
I understand. So for now I'll leave a comment on PR or make an Issure if I notice something about translations of Kotohogushi.

**Comment by Tymaeus** [comment]
> Please wait a little while before touching Kotohogushi. I need to do some stuff to the script files and I was planning to do a review of my own. I want to avoid dealing with merge conflicts from 3 people editing the same files at the same time.

On that note, what should I do with the translation of the rest of Kotohogushi, since you're planning to do script work? Keeping going as is (after the latest patch is released, I mean), or wait?

**Comment by DoctorDiablo** [comment]
> On that note, what should I do with the translation of the rest of Kotohogushi, since you're planning to do script work? Keeping going as is (after the latest patch is released, I mean), or wait?

You can keep going like you've been doing. I'm going to just accept your Koto pull requests as-is so that I can easily work on top of your changes. And I'll try to avoid touching the files you haven't gotten to yet.

Hopefully I can put together a release tonight that rolls up all the latest fixes.

**Comment by Tymaeus** [comment]
> > On that note, what should I do with the translation of the rest of Kotohogushi, since you're planning to do script work? Keeping going as is (after the latest patch is released, I mean), or wait?
> 
> You can keep going like you've been doing. I'm going to just accept your Koto pull requests as-is so that I can easily work on top of your changes. And I'll try to avoid touching the files you haven't gotten to yet.
> 
> Hopefully I can put together a release tonight that rolls up all the latest fixes.

Got it, I'll wait on that release, then.

File: Update/omot_009.txt [hunk 1]
```diff
@@ -545,7 +545,7 @@ void main()
 //r梨花ちゃんから聞かされた、去年叔父夫婦と暮らしていたときの沙都子の生活は、酷いと一言では片付けられないほどのものだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "梨花ちゃんから聞かされた、去年叔父夫婦と暮らしていたときの沙都子の生活は、酷いと一言では片付けられないほどのものだった。",
-			NULL, "According to Rika-chan, Satoko's life with her uncle last year was so awful that it couldn't even be put into words.", Line_Normal);
+			NULL, "According to Rika-chan, Satoko's life with her uncle last year was beyond the word awful.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_009.txt:548]:
```suggestion
			NULL, "According to Rika-chan, Satoko's life with her uncle last year was so far beyond awful that the word didn't cut it.", Line_Normal);
```

File: Update/omot_009.txt [hunk 2]
```diff
@@ -1150,7 +1150,7 @@ void main()
 //r葛西さんは…とチラリと視線をやると、パフェには一切手をつけず、優雅にブラックコーヒーをすすっていた。…いつの間にコーヒーなんて頼んでいたんだ？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "葛西さんは…とチラリと視線をやると、パフェには一切手をつけず、優雅にブラックコーヒーをすすっていた。…いつの間にコーヒーなんて頼んでいたんだ？",
-			NULL, "Kasai-san... periodically glanced at me while gracefully sipping his coffee, without touching his parfait at all. ...When did he even ask for that coffee?", Line_Normal);
+			NULL, "What's up with Kasai-san...? I peeked at him. He was gracefully sipping his coffee, without touching his parfait at all. ...When did he even ask for that coffee?", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_009.txt:1153]:
```suggestion
			NULL, "When I glanced at Kasai-san... he was gracefully sipping his coffee, without touching his parfait at all. ...When did he even ask for that coffee?", Line_Normal);
```

File: Update/omot_009_choice4.txt [hunk 3]
```diff
@@ -138,7 +138,7 @@ void main()
 			NULL, "\"Yes, well...", Line_WaitForInput);
 	ModPlayVoiceLS(4, 15, "ps3/s14/15/2115000039", 256, TRUE);
 	OutputLine(NULL, "ですが、クラブの若い衆の何人かと妙につるんだりして、金もないだろうに変だ、と思っていたところなんです」",
-			NULL, " Even with her lack of money, she's been strangely popular with some of the younger club members.\"", Line_Normal);
+			NULL, " However, even with her lack of money, she's been strangely popular with some of the younger club members. I've been thinking that's weird.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_009_choice4.txt:141]:
```suggestion
			NULL, " However, even with her lack of money, she's been strangely popular with some of the younger club members. I've been thinking that's odd.\"", Line_Normal);
```

File: Update/omot_009_togoodend.txt [hunk 4]
```diff
@@ -254,7 +254,7 @@ void main()
 			NULL, "\"Yes, well...", Line_WaitForInput);
 	ModPlayVoiceLS(4, 15, "ps3/s14/15/2115000062", 256, TRUE);
 	OutputLine(NULL, "そりゃ数十分もあれば…」",
-			NULL, " that would likely take just an hour or two...\"", Line_Normal);
+			NULL, " that would likely take just several tens of minutes...\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_009_togoodend.txt:257]:
```suggestion
			NULL, " that would likely take just half an hour or so...\"", Line_Normal);
```

**Review by DoctorDiablo — APPROVED** [review]

**Review by DoctorDiablo — APPROVED** [review]

File: Update/omot_009.txt [hunk 5]
```diff
@@ -1233,7 +1233,7 @@ void main()
 //r詩音の冗談にも、口元だけの微笑で全く動じたそぶりを感じられない。付き合いの長さもあるだろうけど、それ以上に男としての懐の深さと強さがうかがえた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "詩音の冗談にも、口元だけの微笑で全く動じたそぶりを感じられない。付き合いの長さもあるだろうけど、それ以上に男としての懐の深さと強さがうかがえた。",
-			NULL, "Even after Shion's joke, I couldn't detect even a hint of a smile out of him. They seemed to have a long relationship together, but more than that I was able to see the depth and strength of him as a man.", Line_Normal);
+			NULL, "Even after Shion's joke, I couldn't detect any reaction out of him except his lip with a slight smile. They seemed to have a long relationship together, but more than that I was able to see the depth and strength of him as a man.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_009.txt:1236]:
```suggestion
			NULL, "Even after Shion's joke, I couldn't detect any reaction out of him except a slight smile on his lips. They seemed to have a long relationship together, but more than that I was able to see the depth and strength of him as a man.", Line_Normal);
```

**Review by DoctorDiablo — APPROVED** [review]

**Pull request merged**